<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3DWindow</class>
 <widget class="QMainWindow" name="Q3DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>602</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Live Exporter - Qgis2threejs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Q3DView" name="webView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_to_Web"/>
     <addaction name="separator"/>
     <addaction name="actionExport_as_Image"/>
     <addaction name="actionSTL_format"/>
    </widget>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuCamera">
     <property name="title">
      <string>Camera</string>
     </property>
     <addaction name="actionPerspective"/>
     <addaction name="actionOrthogonal"/>
    </widget>
    <widget class="QMenu" name="menuControls">
     <property name="title">
      <string>Controls</string>
     </property>
     <addaction name="actionOrbit"/>
     <addaction name="actionTrackball"/>
    </widget>
    <widget class="QMenu" name="menuPanel">
     <property name="title">
      <string>Panel</string>
     </property>
     <addaction name="actionLayer_Panel"/>
    </widget>
    <addaction name="actionWorld_Settings"/>
    <addaction name="menuCamera"/>
    <addaction name="menuControls"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Camera_Position"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="menuPanel"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionAlways_on_Top"/>
   </widget>
   <widget class="QMenu" name="menuLayer">
    <property name="title">
     <string>&amp;Layer</string>
    </property>
    <addaction name="actionAdd_Layer"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuLayer"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Layer"/>
   <addaction name="separator"/>
   <addaction name="actionExport_to_Web"/>
   <addaction name="separator"/>
   <addaction name="actionReset_Camera_Position"/>
   <addaction name="actionReload"/>
  </widget>
  <action name="actionSTL_format">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export as STL format</string>
   </property>
  </action>
  <action name="actionWorld_Settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>World Settings...</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Orthographic</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAlways_on_Top">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always on Top</string>
   </property>
  </action>
  <action name="actionExport_to_Web">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export to Web...</string>
   </property>
  </action>
  <action name="actionExport_as_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export as Image</string>
   </property>
  </action>
  <action name="actionReset_Camera_Position">
   <property name="text">
    <string>Reset Camera Position</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionOrbit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orbit</string>
   </property>
  </action>
  <action name="actionTrackball">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trackball</string>
   </property>
  </action>
  <action name="actionAdd_Layer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Layer...</string>
   </property>
  </action>
  <action name="actionLayer_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Layer Panel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3DView</class>
   <extends>QWidget</extends>
   <header>q3dview</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
